connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroOptions.scala

```diff
@@ -41,6 +41,12 @@ private[sql] class AvroOptions(
     this(CaseInsensitiveMap(parameters), conf)
   }
 
+  private def getString(paramName: AvroOptions.Value): Option[String] = {
+    parameters.get(paramName.toString)
+  }
```

Comment:
Seems like this leads to a much higher line of changes, don't think it adds much value